src_python/habitat_sim/robots/mobile_manipulator.py

```diff
@@ -274,15 +274,17 @@ def reset(self) -> None:
     @property
     def arm_joint_limits(self) -> Tuple[np.ndarray, np.ndarray]:
         """Get the arm joint limits in radians"""
-        arm_pos_indices = list(
-            map(lambda x: self.joint_pos_indices[x], self.params.arm_joints)
-        )
 
+        # deref self vars to cut access in half
+        joint_pos_indices = self.joint_pos_indices
+        joint_limits = self.joint_limits
```

Comment:
```suggestion
        lower_joints_limits, upper_joint_limits = self.joint_limits
```

Reply:
Right on.  Now we're thinking with portals :D